html/semantics/interactive-elements/the-dialog-element/backdrop-receives-element-events.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<title>Test that ::backdrop receives click events for the associated element</title>
+<link rel="author" title="Tim Nguyen" href="https://github.com/nt1m">
+<body>
+<style>
+/* ::backdrop takes up whole screen, actual <dialog> is hidden */
+dialog {
+    visibility: hidden;
+    pointer-events: none;
+}
+
+dialog::backdrop {
+    background-color: red;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+}
+
+dialog.clicked::backdrop {
+    background-color: green;
+}
+</style>
+<dialog></dialog>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-actions.js"></script>
+<script>
+const dialog = document.querySelector("dialog");
+dialog.showModal();
+async_test(function() {
+    dialog.addEventListener("click", () => {
+        dialog.className = "clicked";
+        this.done();
```

Comment:
This is what makes the test pass, not the red/green color change. Can you add a comment earlier that the color is just for debugging, and has no effect on the test?

Reply:
Done!